CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ All notable changes to this project will be documented in this file. `Jayme` a
 
 ## 2.0.1
 
+- Respotory Create Method now sends the ID in the HTTP Parameters instead of doing it in the URL. (Issue [#63](https://github.com/inaka/Jayme/issues/63))
```

Comment:
Change it for:
- `create(...)` function in `CRUD` repository no longer attachs the `:id` parameter in its `POST` URL. (Issue [#63](https://github.com/inaka/Jayme/issues/63))